EnglishTranslation-master/QUEST_LV_0300.tsv

```diff
@@ -1657,16 +1657,16 @@ QUEST_LV_0300_20160310_001656	You've defeated Spions that came after smelling th
 QUEST_LV_0300_20160310_001657	A great success or a great failure
 QUEST_LV_0300_20160310_001658	It seems that the reagent has been completed. Talk with Lucien Winterspoon.
 QUEST_LV_0300_20160310_001659	Test Lucien's reagent on the tree vines that are in the tent of herb pickers.
-QUEST_LV_0300_20160310_001660	Lucien Winterspoon told you that the tree vines that are growing in the herb broker��s tent are good to test for the reagent. For the test of Lucien��s reagent, please move to the tent of the herb collector.
+QUEST_LV_0300_20160310_001660	Lucien Winterspoon told you that the tree vines that are growing in the herb broker's tent are good to test for the reagent. For the test of Lucien's reagent, please move to the tent of the herb collector.
 QUEST_LV_0300_20160310_001661	As you used the reagent, the tree vines grew irregularly and soon they broke. Talk with Lucien Winterspoon.
 QUEST_LV_0300_20160310_001662	The tree vines grew at an enormously fast pace and then they broke. Lucien Winterspoon is happy since she thinks that her experiment is successful. Talk to her.
 QUEST_LV_0300_20160310_001663	Receive advices from Bidotas
-QUEST_LV_0300_20160310_001664	Lucien Winterspoon seems she��s mad after listening from Bidotas to get advices.  She told you to go meet Bidotas alone if she really wants to see Bidotas.
+QUEST_LV_0300_20160310_001664	Lucien Winterspoon seems she's mad after listening from Bidotas to get advices.  She told you to go meet Bidotas alone if she really wants to see Bidotas.
```

Comment:
`Lucienne Winterspoon seems she's mad after listening from Vaidotas to get advice.  She told you to go meet Vaidotas alone if she really wants to see him.`